api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/pac4j/saml/Pac4jSamlClientProperties.java

```diff
@@ -169,12 +170,41 @@
      */
     private boolean signServiceProviderMetadata;
 
+    /**
+     * Whether or not the authnRequest should be signed.
+     */
+    private boolean authnRequestSigned;
+
+    /**
+     * Whether or not the Logout Request sent from the SP should be signed.
+     */
+    private boolean spLogoutRequestSigned;
```

Comment:
Change to `signServiceProviderLogoutRequest`

Reply:
Done